### PR TITLE
remove account from device group

### PR DESCRIPTION
**Pull request by mgold1234**
depend on remove account from device
related to THEEDGE-2333

Signed-off-by: mgold1234 <mgold@redhat.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/services/devices.go [hunk 1]
```diff
@@ -82,10 +82,10 @@ type PlatformInsightsDeleteEventPayload struct {
 // NewDeviceService gives a instance of the main implementation of DeviceServiceInterface
 func NewDeviceService(ctx context.Context, log *log.Entry) DeviceServiceInterface {
 	return &DeviceService{
-		UpdateService: NewUpdateService(ctx, log),
-		ImageService:  NewImageService(ctx, log),
-		Inventory:     inventory.InitClient(ctx, log),
-		Service:       Service{ctx: ctx, log: log.WithField("service", "image")},
+		//	UpdateService: NewUpdateService(ctx, log),
```
Comment (ldjebran) [inline pkg/services/devices.go:85]: why you remove this service

**Review by ldjebran — COMMENTED** [review]
Looks good , one comments , and resolve conflicts

File: pkg/models/devicegroups_test.go [hunk 2]
```diff
@@ -14,11 +14,12 @@ func TestGroupValidateRequest(t *testing.T) {
 		group    *DeviceGroup
 		expected error
 	}{
-		{name: "Empty name", group: &DeviceGroup{OrgID: "222222", Account: "111111", Type: "static"}, expected: errors.New(DeviceGroupNameEmptyErrorMessage)},
-		{name: "Invalid type", group: &DeviceGroup{Name: "test_group", OrgID: "222222", Account: "111111", Type: "invalid type"}, expected: errors.New(DeviceGroupTypeInvalidErrorMessage)},
-		{name: "Invalid name", group: &DeviceGroup{Name: "** test group", OrgID: "222222", Account: "111111", Type: DeviceGroupTypeDefault}, expected: errors.New(DeviceGroupNameInvalidErrorMessage)},
-		{name: "Empty account", group: &DeviceGroup{Name: "test_group", Type: "static", OrgID: "222222"}, expected: errors.New(DeviceGroupAccountEmptyErrorMessage)},
-		{name: "Valid DeviceGroup", group: &DeviceGroup{Name: "test_group", Account: "111111", OrgID: "222222", Type: DeviceGroupTypeDefault}, expected: nil},
+
```
Comment (AaronH88) [inline pkg/models/devicegroups_test.go:17]: is this empty line needed?

File: pkg/routes/devicegroups.go [hunk 3]
```diff
@@ -671,8 +670,7 @@ func UpdateAllDevicesFromGroup(w http.ResponseWriter, r *http.Request) {
 	commitID, err := ctxServices.DeviceService.GetLatestCommitFromDevices(orgID, setOfDeviceUUIDS)
 	if err != nil {
 		ctxServices.Log.WithFields(log.Fields{
-			"error":   err.Error(),
-			"account": account,
+			"error": err.Error(),
```
Comment (AaronH88) [inline pkg/routes/devicegroups.go:673]: lets also log the org_id in this error case.
`"org_id": ordID,`

File: pkg/routes/devicegroups.go [hunk 4]
```diff
@@ -683,26 +681,23 @@ func UpdateAllDevicesFromGroup(w http.ResponseWriter, r *http.Request) {
 	commit, err := ctxServices.CommitService.GetCommitByID(devicesUpdate.CommitID)
 	if err != nil {
 		ctxServices.Log.WithFields(log.Fields{
-			"error":   err.Error(),
-			"account": account,
+			"error": err.Error(),
```
Comment (AaronH88) [inline pkg/routes/devicegroups.go:684]: Same here

File: pkg/routes/devicegroups.go [hunk 5]
```diff
@@ -683,26 +681,23 @@ func UpdateAllDevicesFromGroup(w http.ResponseWriter, r *http.Request) {
...
 		return
 	}
 	// should be refactored to avoid performance issue with large volume
-	updates, err := ctxServices.UpdateService.BuildUpdateTransactions(&devicesUpdate, account, orgID, commit)
+	updates, err := ctxServices.UpdateService.BuildUpdateTransactions(&devicesUpdate, orgID, commit)
 	if err != nil {
 		ctxServices.Log.WithFields(log.Fields{
-			"error":   err.Error(),
-			"account": account,
+			"error": err.Error(),
```
Comment (AaronH88) [inline pkg/routes/devicegroups.go:693]: same here

File: pkg/routes/devicegroups.go [hunk 6]
```diff
@@ -683,26 +681,23 @@ func UpdateAllDevicesFromGroup(w http.ResponseWriter, r *http.Request) {
...
 		}).Error("Error building update transaction")
 		respondWithAPIError(w, ctxServices.Log, errors.NewInternalServerError())
 		return
 	}
 	// should be refactored to avoid performance issue with large volume
 	var upd []models.UpdateTransaction
 	for _, update := range *updates {
```
Comment (AaronH88) [inline pkg/routes/devicegroups.go:700]: Lets set the org_id here

`update.OrgID = orgID`

File: pkg/services/devicegroups_test.go [hunk 7]
```diff
@@ -377,27 +377,28 @@ var _ = Describe("DeviceGroupsService basic functions", func() {
...
-				deletedDevices, delErr := deviceGroupsService.DeleteDeviceGroupDevices(account, orgID, deviceGroupID, fakeDeviceGroup.Devices)
+				deletedDevices, delErr := deviceGroupsService.DeleteDeviceGroupDevices(orgID, deviceGroupID, fakeDeviceGroup.Devices)
 				Expect(delErr).NotTo(BeNil())
 				Expect(delErr.Error()).To(Equal(new(services.DeviceGroupDevicesNotFound).Error()))
 				Expect(deletedDevices).To(BeNil())
 			})
 
 			It("should return error when device does not exist in device-group", func() {
-				_, err := deviceGroupsService.DeleteDeviceGroupDevices(account, orgID, deviceGroupID, []models.Device{savedDeviceGroup.Devices[0]})
+				_, err := deviceGroupsService.DeleteDeviceGroupDevices(
```
Comment (AaronH88) [inline pkg/services/devicegroups_test.go:387]: any reason this was split over two lines?

**Review by ldjebran — APPROVED** [review]
LGTM

**Review by ldjebran — APPROVED** [review]
LGTM

**Pull request merged**